### DISCUSSION TITLE
How to use

### DISCUSSION THREAD

**Discussion by Mariete124** [vzhd1701/csv2notion#1] [Q&A]
Hi I have tried everything. I'v install the package, I've download the info but I'm not able to change the info of the exe o to run the code in the cmd or python. I'm a bit lost :)

**Comment by vzhd1701** [comment]
Hi. What OS are you using?

**Reply by Mariete124** [reply]
Hi, I'm working with windows 10. Thanks for everything by the way

**Reply by vzhd1701** [reply]
Have you tried using [pre-compiled](https://github.com/vzhd1701/csv2notion/releases/download/v0.2.0/bin_csv2notion_0.2.0_win_x64.zip) binary?

**Reply by Mariete124** [reply]
I've tryed to usé te precompiled app with the exe and running the Code on the consol

**Reply by Mariete124** [reply]
But I don't know how to edit the edit of the exe

**Reply by vzhd1701** [reply]
You don't need to edit the exe file. Just open [command line in a directory](https://www.thewindowsclub.com/how-to-open-command-prompt-from-right-click-menu) where `csv2notion.exe` is located, and then type `csv2notion.exe` in the command line and press enter.

**Reply by Mariete124** [reply]
Yeaaaaaaa that work!!! You are amazing, now the thing is that is adding the csv line instead of merg it with the one already exisitng. 
![imagen_2022-05-25_184634214](https://user-images.githubusercontent.com/94454662/170317143-47f38bb3-6472-40a6-9e16-73a3254998b4.png)
So I have that row. 
![image](https://user-images.githubusercontent.com/94454662/170317442-5e1c8a92-4be7-4cb9-9bfd-0dcfbf9822e6.png)
And this csv
![image](https://user-images.githubusercontent.com/94454662/170447543-3eb4c8dc-6a4a-43d0-8f8c-6e0f10604d98.png)
I want to update de In state from off to ingest.
Now when I run it
```bash
csv2notion --token TOKEN --url URL --merge data.csv
```
It create another line instead of update the existing one, is there a way to update instead of create a new one? Thanks for everything
![image](https://user-images.githubusercontent.com/94454662/170447823-b355484a-f5ad-4730-9d54-c8fdd4f0fd96.png)

**Reply by vzhd1701** [reply]
The key column in CSV must have the same name it has in Notion DB. You must change "Key" to "Name" in CSV for merge to work.

Also, you should not share your token! It gives access to your account to anyone who has it. I edited your post to hide it, but you should re-login your account to disable this token to be safe.

**Reply by Mariete124** [reply]
Ouuu thanks for the advice I didn't know it was so dangerous. I've tried it but I still have the same issue
![image](https://user-images.githubusercontent.com/94454662/170482294-1b43a874-dad2-4ae0-b78e-e121ea8ee6d6.png)

![image](https://user-images.githubusercontent.com/94454662/170481802-f8d944d8-98cc-45ed-a431-94c9a0b17aaa.png)
40d8-bb95-52965cd944bb.png)
Seriously thanks for everythin, I've been weeks trying to acomplish this and I feel I'm getting closer with this

**Reply by vzhd1701** [reply]
Hmm, this is strange. Could you please share the csv file and a link to your database?

Make sure that you allow making duplicate so I could import it into my workspace for testing

![image](https://user-images.githubusercontent.com/5212766/170483084-bb5d186a-21f2-4bb0-a64a-9557795f44e7.png)

**Reply by Mariete124** [reply]
Yea sure I share it with u:

[prueba.csv](https://github.com/vzhd1701/csv2notion/files/8779301/prueba.csv)

https://caring-joggers-e18.notion.site/2860569e9f304080b0b8a3a865f46146?v=85448385d13445ec9b3ce313856e3f5f

**Reply by vzhd1701** [reply]
I think I see what the problem is. You've got extra space after `SHLL_105_002_0010_PL01_v001` in the CSV file.

**Reply by Mariete124** [reply]
YEAAAAAAAAAAA THAT WAS ITTTTTT. OMG Man this work is amazing thank so much realy!!!!!!

**Reply by vzhd1701** [reply]
No problem! Happy to hear that it works for you now :)
